src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -101,6 +113,19 @@ public void removePerson(Person key) {
         persons.remove(key);
     }
 
+    public void addAppointment(Appointment appointment) {
```

Comment:
this is rlly nitpicky, but java convention usually has add return a boolean value to indicate if the addition was successful